ui/Screen/Profile/Projects.svelte

```diff
@@ -1,5 +1,26 @@
 <script>
-  import { ProjectList } from "../../DesignSystem/Component";
+  import { push } from "svelte-spa-router";
+
+  import * as path from "../../lib/path.js";
+  import { projectNameStore, projects as store } from "../../src/project.ts";
+
+  import { Text } from "../../DesignSystem/Primitive";
+  import { ProjectList, Remote } from "../../DesignSystem/Component";
+
+  const select = event => {
+    const project = event.detail;
+
+    projectNameStore.set(project.metadata.name);
+    push(path.projectSource(project.id));
+  };
 </script>
 
-<ProjectList />
+<Remote {store} let:data={projects}>
+  {#if projects.length > 0}
+    <ProjectList {projects} on:select={select} />
+  {:else}{push(path.profileOnboard())}{/if}
```

Comment:
I wonder if prettier did this?

Reply:
Think so, buuuh prettier. I think we should put some love in the svelte plugin.